S2VX.Game/Story/Command/S2VXCommand.cs

```diff
@@ -55,11 +55,15 @@ public abstract class S2VXCommand : IComparable<S2VXCommand> {
             var commandName = split[0];
             var systemType = Type.GetType($"S2VX.Game.Story.Command.{commandName}Command");
             var staticMethod = systemType.GetMethod("FromString", BindingFlags.Public | BindingFlags.Static);
-            var command = staticMethod.Invoke(null, new object[] { split }) as S2VXCommand;
-            command.StartTime = double.Parse(split[1], CultureInfo.InvariantCulture);
-            command.EndTime = double.Parse(split[2], CultureInfo.InvariantCulture);
-            command.Easing = Enum.Parse<Easing>(split[3]);
-            return command;
+            try {
+                var command = staticMethod.Invoke(null, new object[] { split }) as S2VXCommand;
+                command.StartTime = double.Parse(split[1], CultureInfo.InvariantCulture);
+                command.EndTime = double.Parse(split[2], CultureInfo.InvariantCulture);
+                command.Easing = Enum.Parse<Easing>(split[3]);
+                return command;
+            } catch (TargetInvocationException) {
+                return null;
```

Comment:
I don't think we should put try catch in this function

Reply:
If we only handle errors when adding new commands from the CommandPanel, then it'd still crash if someone added a command manually by editing the .s2ry right?

Reply:
Yeah, I would add another try catch block when loading the story. I'm not sure of the best practice for exception handling, but it seems pretty awkward if we try and catch errors in S2VXCommand itself because S2VXCommand wouldn't really know how to properly handle the exception, e.g. rolling back changes or showing the error indicator to user.